games/__init__.py

```diff
@@ -87,4 +101,30 @@
                 data_provider_game_id='dnb',
                 data_provider_variant_id=-1)
         }),
+
+    'tilechess': Game(
+        name='Tile Chess',
+        desc='A chess variant (with slightly different valid moves). Wins when checkmate opponents King',
+        variants={
+            'regular': GameVariant(
+                name='Regular',
+                desc='Regular',
+                data_provider=GamesmanClassicDataProvider,
+                data_provider_game_id='tilechess',
+                data_provider_variant_id=-1)
+        }),
+
+    '1ton': Game(
```

Comment:
Since we already have `1210` for 0..10 by 1 or 2, we can perhaps combine `1210` and `1ton` together in our API? We can probably also use some variants of `1ton` to provide solutions to 0..50?! by 1 or 2. So the variant ID we output could be `10` for 0..10 and `50` for 0..50?